admin_guide/managing_projects.adoc

```diff
@@ -90,39 +90,69 @@ to all authenticated developers by default.
-Removing the `self-provisioners`
-xref:../architecture/additional_concepts/authorization.adoc#roles[cluster role]
-from authenticated user groups will deny permissions for self-provisioning any new projects.
 
-----
-$ oc adm policy remove-cluster-role-from-group self-provisioner system:authenticated system:authenticated:oauth
-----
-
-When disabling self-provisioning, set the `projectRequestMessage` parameter in the
-*_master-config.yaml_* file to instruct developers on how to request a new
-project. This parameter is a string that will be presented to the developer in
-the web console and command line when they attempt to self-provision a project.
-For example:
+You can prevent an authenticated user group from self-provisioning new projects.
 
+. Log in as a user with
+xref:../architecture/additional_concepts/authorization.adoc#roles[*cluster-admin*]
+privileges.
+. Remove the `self-provisioners`
+xref:../architecture/additional_concepts/authorization.adoc#roles[cluster role]
+from the group.
++
 ----
-Contact your system administrator at projectname@example.com to request a project.
+$ oc adm policy remove-cluster-role-from-group self-provisioner system:authenticated system:authenticated:oauth
 ----
 
-or:
-
-----
-To request a new project, fill out the project request form located at
+. Set the `projectRequestMessage` parameter value in the
+*_master-config.yaml_* file to instruct developers how to request a new
+project. This parameter value is a string that will be presented to a user in
+the web console and command line when the user attempts to self-provision a project.
+You might use one of the following messages:
++
+* To request a project, contact your system administrator at 
+projectname@example.com.
+* To request a new project, fill out the project request form located at
 https://internal.example.com/openshift-project-request.
-----
 
++
 .Example YAML file
+[source,yaml]
 ----
 ...
 projectConfig:
   ProjectRequestMessage: "message"
   ...
 ----
 
+. Edit the `self-provisioners` cluster role binding to prevent 
+xref:../upgrading/manual_upgrades.adoc#updating-policy-definitions[automatic updates]
+to the role. Automatic updates reset the cluster roles to the default state.
+** To update the role binding from the command line:
+... Run the following command:
++
+----
+$ oc edit clusterrolebinding.rbac self-provisioners
+----
+... In the displayed role binding, set the `rbac.authorization.kubernetes.io/autoupdate` parameter
+ value to `false`, as shown in the following example:
++
+[source,yaml]
+----
+apiVersion: authorization.openshift.io/v1
+kind: ClusterRoleBinding
+metadata:
+  annotations:
+    rbac.authorization.kubernetes.io/autoupdate: "false"
+...
+----
+
+ ** To update the role binding by using automation, use the following command:
++
+----
+$ oc patch clusterrolebinding.rbac self-provisioners -p '{ "metadata": { "annotations": { "rbac.authorization.kubernetes.io/autoupdate": "false" } } }'
+----
```

Comment:
Not sure about the "by using automation" line. Makes me think of Ansible. 
How about ```

To update the role binding, edit the *self-provisioners* cluster role binding:
:...
Or,  use the following command:
```

Reply:
That's fair. I think it is mean to be dropped in your own automated scripts.